modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -122,6 +124,8 @@ impl ServerModuleGen for WalletGen {
                     params.consensus.network,
                     params.consensus.finality_delay,
                     params.local.bitcoin_rpc.clone(),
+                    // TODO: make configurable
```

Comment:
It seems this is now configurable by environment variable?

Reply:
What do you mean by that? I don't think I exposed that anywhere so far since I see including this in the server cfg as a temporary fix only. We need some modularized way of letting clients define their own local config that they don't receive from the server imo.

Reply:
Yea nevermind. `default_esplora_server` just sets default, doesn't read them from environment.

Reply:
*If* we want to keep it this way it should be a module config gen parameter.